lib/advancement_conditions/dual_round_condition.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module AdvancementConditions
+  class DualRoundCondition < AdvancementCondition
+    def self.wcif_type
+      "dual"
+    end
+
+    def to_s(_round, short: false)
+      I18n.t("advancement_condition#{'.short' if short}.dual")
+    end
+
+    def max_advancing(results)
+      results.count
+    end
```

Comment:
Depends on how you want to use this.
At this early prototyping stage, my thoughts are more along the lines of...
- R1 and R2 are linked as a "dual round"
- R1 -> R2 has a percent advancement criterion, with 100% being set
- R2 -> whatever comes after it is the one that has the dual condition
 
In essence, the dual round condition is not the one between the dual rounds themselves, but rather the one at the end marking "hey this _was_ a dual round, you need to consider both rounds".
With this idea in mind, the total number of results is then *not* correct for `max_advancing`